### PR TITLE
Functionnal backlist system

### PR DESCRIPTION
**Pull request by immjs**
Pls pull ;-;

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -12,13 +12,18 @@ const config = JSON.parse(
 bot.config = config;
 bot.commands = new Commands(bot);
 bot.logger = new Logger();
-bot.blacklist = new Discord.Collection();
```
Comment (nobobo1234) [inline src/bot.js:15]: My actual goal with this was to add a list of users that are abusing the bot so they can't use it, but this is also a great idea!

File: src/bot.js [hunk 2]
```diff
@@ -12,13 +12,18 @@ const config = JSON.parse(
...
     const eventFunc = require(path.join(__dirname, 'events', name));
     bot.on(name, (...args) => eventFunc.run(bot, ...args));
 }
-
+bot.on('message',message=>{
+    if(bot.blacklist.some(a=>message.content.toLowerCase().includes(a.toLowerCase()))){
+        message.delete(500);//In case this is triggered before the message is actually sent, so that way there is no undeletable messages (I have experience with this problem)
+        message.channel.send(message.author.toString()+" Hi! Please don't make use of obscene/ discriminatory words, thanks!")
+    }
+})
```
Comment (nobobo1234) [inline src/bot.js]: Please add this to the command manager in the `handleCommand()` function. You should take a look into the structure of the bot before adding something :wink:

**Review by nobobo1234 — COMMENTED** [review]
Please change the thing I mentioned

File: src/bot.js (hunk 2 above)
Comment (nobobo1234) [reply src/bot.js]: Also for the problem you mentioned, that can be solved by using async await

File: src/events/message.js [hunk 3]
```diff
@@ -1,6 +1,10 @@
 exports.run = (bot, message) => {
     const { prefix } = bot.config;
     if (!message.content.startsWith(prefix)) return;
-
+    if(bot.blacklist.some(a=>message.content.toLowerCase().includes(a.toLowerCase()))){
+        message.delete(500);//In case this is triggered before the message is actually sent, so that way there is no undeletable messages (I have experience with this problem)
+        return message.channel.send(message.author.toString()+" Hi! Please don't make use of obscene/ discriminatory words, thanks!")
+    }
+
```
Comment (nobobo1234) [inline src/events/message.js:8]: As I said, please do this in the commands.handleCommand() function, not here :wink:

**Review by nobobo1234 — CHANGES_REQUESTED** [review]

**Comment by immjs** [comment]
I don't get how 🤔

**Comment by nobobo1234** [comment]
`src/managers/commands.js` is the place where the command is handled, please add your code there

**Comment by immjs** [comment]
Ok I think I get it, will do this when I get my computer

**Comment by immjs** [comment]
Moving to my other PR.

**Pull request closed without merging**